tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -0,0 +1,110 @@
+        custom_feed_path = custom_alas_json_feed_path
+    elif alas_feed_path == custom_alas2_json_feed_path:
+        custom_feed_path = custom_alas2_json_feed_path
+
+    backup_data = read_json_file(custom_feed_path)
+    modified_data = deepcopy(backup_data)
+
+    # Insert key:value pair as string, since otherwise, you could not insert lists or dictionaries as a key
+    modified_string_data = vd.insert_data_json_feed(data=modified_data, field_name=test_values[0],
+                                                    field_value=test_values[1], append_data=None, brackets=False)
+
+    write_file(custom_feed_path, modified_string_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    control_service('restart', daemon='wazuh-modulesd')
+
+    vd.set_system(system='ALAS')
+
+    yield
+
+    write_json_file(custom_feed_path, backup_data)
+
+    vd.clean_vuln_and_sys_programs_tables()
+
+    truncate_file(LOG_FILE_PATH)
+
+
+def test_no_feed_changes(clean_vuln_tables, get_configuration, configure_environment, restart_modulesd):
+    """Check if the feed is imported successfully by default."""
+    if get_configuration['metadata']['alas_custom_feed'] == custom_alas_json_feed_path:
+        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS_LOG,
+                                            expected_vulnerabilities_number=vd.ALAS_NUM_CUSTOM_VULNERABILITIES)
+    elif get_configuration['metadata']['alas_custom_feed'] == custom_alas2_json_feed_path:
+        vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS2_LOG,
+                                            expected_vulnerabilities_number=vd.ALAS2_NUM_CUSTOM_VULNERABILITIES)
+
+
+@pytest.mark.parametrize('test_values', vd.EXTRA_TEST_VALUES, ids=vd.EXTRA_TEST_IDS)
+def test_extra_tags_alas_linux_feed(test_values, clean_vuln_tables, get_configuration, configure_environment,
+                                    modify_feed):
+    """Check if Vulnerability Detector behaves as expected while importing Amazon Linux JSON feed with extra tags."""
+    field = test_values[0]
+
+    if type(field) in [str]:
+        if get_configuration['metadata']['alas_custom_feed'] == custom_alas_json_feed_path:
+            vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS_LOG,
+                                                expected_vulnerabilities_number=vd.ALAS_NUM_CUSTOM_VULNERABILITIES)
+        elif get_configuration['metadata']['alas_custom_feed'] == custom_alas2_json_feed_path:
+            vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS2_LOG,
+                                                expected_vulnerabilities_number=vd.ALAS2_NUM_CUSTOM_VULNERABILITIES)
+    else:
+        vd.check_failure_when_importing_feed(wazuh_log_monitor=wazuh_log_monitor, parser_error=True)
+
+    vd.check_if_modulesd_is_running()
```

Comment:
```suggestion
@pytest.mark.parametrize('test_values', vd.EXTRA_TEST_VALUES, ids=vd.EXTRA_TEST_IDS)
def test_extra_tags_alas_linux_feed(modify_feed, up_required_vd_daemons, test_values, set_system, get_configuration, configure_environment, restart_modulesd_function, file_monitoring):
    """Check if Vulnerability Detector behaves as expected while importing Amazon Linux JSON feed with extra tags."""
    field = test_values[0]

    if type(field) in [str]:
        if get_configuration['metadata']['alas_custom_feed'] == custom_alas_json_feed_path:
            vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS_LOG,
                                                expected_vulnerabilities_number=vd.ALAS_NUM_CUSTOM_VULNERABILITIES)
        elif get_configuration['metadata']['alas_custom_feed'] == custom_alas2_json_feed_path:
            vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ALAS2_LOG,
                                                expected_vulnerabilities_number=vd.ALAS2_NUM_CUSTOM_VULNERABILITIES)
    else:
        vd.check_failure_when_importing_feed(wazuh_log_monitor=wazuh_log_monitor, parser_error=True)

    vd.check_if_modulesd_is_running()

```

Reply:
Done in commit 330f641b6fd3cb999e7d4142cfe95ec1854f80ef and 869101f5e36f1c088568d9b5d3f5e66c465768f4